dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/AbstractSpringTests.java

```diff
@@ -254,6 +255,12 @@ public void configureServer() throws Exception {
                 if (expectWebApplication()) {
                     assertNotNull("The endpoint is not available", server
                                     .waitForStringInLog("CWWKT0016I:.*"));
+
+                    String testMethodName = testName.getMethodName();
+                    if (testMethodName != null && testMethodName.contains(DEFAULT_HOST_WITH_APP_PORT)) {
```

Comment:
In the DEFAULT_HOST_WITH_APP_PORT case do we still need the check above for `.waitForStringInLog("CWWKT0016I:.*")`?

Reply:
It is doing the same check twice, i will change that